recipe/meta.yaml

```diff
@@ -64,6 +64,8 @@ test:
     - blas =*=*{{ blas_impl }}
   commands:
     - f2py -h
+    # show SIMD features; change to np.show_config() for v>=1.22.0
+    - python -c 'import numpy; numpy._pytesttester._show_numpy_info()'
```

Comment:
I think windows [chokes](https://dev.azure.com/conda-forge/feedstock-builds/_build/results?buildId=344546&view=logs&j=171a126d-c574-5c8c-1269-ff3b989e923d&t=1183ba29-a0b5-5324-8463-2a49ace9e213&l=3792) on single quotes?

Reply:
Ehgads this is annoying. I copied from the [output](https://github.com/conda-forge/numpy-feedstock/pull/235#issuecomment-849177266) not the [source](https://github.com/conda-forge/numpy-feedstock/pull/235#issuecomment-849167449)